shared/general/container-ssh.md

```diff
@@ -32,6 +32,8 @@ $ {{ $names.company.short }} ssh <device-uuid> main
 
 This also works in multicontainer applications, simply pass the name of the appropriate service (as defined in docker-compose.yml) instead of `main`.
 
+__Note:__ To run a command in a non-interactive way, you can pipe commands to the CLI's stdin. For example, `echo "uptime; exit;" | balena ssh <device-uuid>`.
```

Comment:
is this new?

Reply:
Just new to docs. It was included in the [newsletter](https://mailchi.mp/balena/8979hnwafn) and there was an internal request to add it (the FD link from the related issue seems to be broken)